packages/react-components/src/types/OnRender.ts

```diff
@@ -43,6 +43,9 @@ export type CustomAvatarOptions = {
    * It has '?' in place of initials, with static font and background colors
    */
   showUnknownPersonaCoin?: boolean;
+  /** Active flag to display border around coin */
+  /* @conditional-compile-remove(raise-hand) */
+  isActive?: boolean;
```

Comment:
if we are to make this generic for the override should we also be making a way to pick the colour?